src/simulation/actions.jl

```diff
@@ -98,9 +98,14 @@ end
 function AutomotiveDrivingModels.propagate{D<:Union{VehicleDef, BicycleModel}}(veh::Entity{VehicleState, D, Int}, action::AccelSteeringDirection, roadway::Roadway, Δt::Float64)
     previousInd = veh.state.posF.roadind
     
-    L = veh.def.a + veh.def.b
-    l = -veh.def.b
-
+    if typeof(veh) == Entity{VehicleState, BicycleModel, Int}
+        L = veh.def.a + veh.def.b
+        l = -veh.def.b
+    elsif typeof(veh) == Entity{VehicleState, VehicleDef, Int}
```

Comment:
You probably mean `elseif`

Reply:
Although I don't think this is the "Julian" way to fix this problem. You should probably have two different functions with different type params, rather than a `Union` if you need a conditional on the types.